Website/Providers/DataProviders/SqlDataProvider/07.04.00.SqlDataProvider

```diff
@@ -691,7 +695,7 @@ BEGIN
         'Reject',
         'Reject',
         NULL,
-        1,
+        2,
```

Comment:
thanks for the feedback, jbrinkman. 
Is there anywere a documentation of workflow, how it is constructed and should be used (it will be difficult to create a UI for it without further background. thank you.